__tests__/app.test.js

```diff
@@ -121,4 +122,56 @@ describe('/api/reviews/:review_id', () => {
       })
     });
   });
-});
\ No newline at end of file
+});
+
+describe('/api/reviews/:review_id/comments', () => {
+  describe('get request', () => {
+    test('get all comments of the review with the given id', () => {
+      return request(app)
+      .get('/api/reviews/3/comments')
+      .expect(200)
+      .then(({body})=>{
+        expect(body.comments).toBeInstanceOf(Array)
+        expect(body.comments).toHaveLength(3)
+
+        body.comments.forEach((comment)=>{
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: expect.any(Number)
```

Comment:
I would certainly assert the exact review_id here - this doesn't yet prove that the comments are for the right one